store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/operation/impl/CreateSplitPointsTest.java

```diff
@@ -18,19 +18,20 @@
 
 import com.google.common.collect.Lists;
 import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.lang3.SystemUtils;
+import org.apache.curator.shaded.com.google.common.io.Files;
```

Comment:
See comments left on `LegacySupportTest.java` about this import and Temporary files.